roles/setup/files/hccm_openshift_usage_lookback_report_generation_query.yaml

```diff
@@ -83,5 +86,5 @@ spec:
       node_capacity_memory_byte_seconds,
       resource_id,
       pod_labels
-    FROM {| .Report.Inputs.ReportName | scheduledReportTableName |}
-    WHERE {| .Report.Inputs.ReportName | scheduledReportTableName |}.report_period_start >= timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart | prestoTimestamp |}' - interval '10' day
+    FROM {| .Report.Inputs.AggregatedReportName | reportTableName |}
```

Comment:
The changes to this file are a) a small but important bugfix, and b) changes to keep up with changes in Operator metering